client/consensus/src/collators.rs

```diff
@@ -17,6 +17,7 @@
 pub mod basic;
 pub mod lookahead;
 
+use cumulus_client_consensus_aura::collators::RelayParentData;
```

Comment:
fmt these, dont want to be super picky but I saw it a couple of tiumes already haha